Conversions/LengthConversion.js

```
@@ -1,268 +1,31 @@
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  return centimeter / 91.44
-}
-
-const yardToCentimeter = (yard) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Centimeter
-  return yard * 91.44
-}
-
-const inchToYard = (inch) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Inch
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  return inch / 36
-}
-
-const yardToInch = (yard) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Inch
-  return yard * 36
-}
-
-const feetToYard = (feet) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Foot_(unit)
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  return feet / 3
-}
-
-const yardToFeet = (yard) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Foot_(unit)
-  return yard * 3
-}
-
-const meterToYard = (meter) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Meter
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  return meter * 1.09361
-}
-
-const yardToMeter = (yard) => {
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Yard
-  // Wikipedia reference: https://en.wikipedia.org/wiki/Meter
-  return yard / 1.09361
+// // This files has functions to convert different length units
+// // Functions take length value as a argument and returns corresponding converted value
+
+const lengthConversion = (length, fromUnit, toUnit) => {
+  // Define a dictionary to map units to meters
+  const meters = {
+    mm: 0.001,
+    cm: 0.01,
+    m: 1,
+    km: 1000,
+    inch: 0.0254,
+    ft: 0.3048,
+    yd: 0.9144,
+    mi: 1609.34
+  }
+
+  // Check if the units are valid
+  if (!(fromUnit in meters) || !(toUnit in meters)) {
+    return 'Invalid units'
```

You should throw an error.